support for desi-tashan.com

hussamnasir opened the pull request:
I worked on one site and have 3 sources working. I would love to maintain one site and keep it working rather that have 10 different sites with ten different sources.

mohit310 committed:
modified for new site.

hussamnasir committed:
Update __init__.py

hussamnasir committed:
Delete apnitv.py

hussamnasir committed:
Delete desitvforum.py

hussamnasir committed:
Update common.py

hussamnasir committed:
Update desirulez.py

hussamnasir committed:
Update desirulez.py

hussamnasir committed:
Optimized code nd removed all sources except Desitvbox.me
The only source active now is Playwire (Flash) Videos
Display only active shows. Completed shows are not visible anymore.

hussamnasir committed:
Added support for my favourite site Desi-Tashan where links always works
Currently on playwire is supported.

hussamnasir committed:
Finished adding desi-tashan site.
Added support for PlayU links, Letuswatch links and PlaywireHD links.
Disabled all other sites. Too much to maintain.
Special thanks to the original authors on whose work this was built (technitude, mohit310 and coder-alpha )

rdudeja1133 commented:
Thanks

Sent from my iPhone

> On Aug 2, 2016, at 4:16 PM, Hussamuddin Nasir notifications@github.com wrote:
> 
> I worked on one site and have 3 sources working. I would love to maintain one site and keep it working rather that have 10 different sites with ten different sources.
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/coder-alpha/DesiTV.bundle/pull/3
> 
> Commit Summary
> 
> modified for new site.
> Update **init**.py
> Delete apnitv.py
> Delete desitvforum.py
> Update common.py
> Update desirulez.py
> Update desirulez.py
> Optimized code nd removed all sources except Desitvbox.me
> Added support for my favourite site Desi-Tashan where links always works
> Finished adding desi-tashan site.
> File Changes
> 
> M Contents/Code/**init**.py (25)
> D Contents/Code/apnitv.py (281)
> M Contents/Code/common.py (6)
> M Contents/Code/desirulez.py (36)
> A Contents/Code/desitashan.py (267)
> M Contents/Code/desitvbox.py (88)
> D Contents/Code/desitvforum.py (284)
> A Contents/Resources/icon-desitashan.png     (0)
> A Contents/Resources/icon-idowatch.png (0)
> A Contents/Resources/icon-playu.png (0)
> M Contents/Services/ServiceInfo.plist (10)
> A Contents/Services/URL/PlayU/ServiceCode.pys (85)
> M Contents/Services/URL/Playwire/ServiceCode.pys (4)
> M Contents/Services/URL/Vidshare/ServiceCode.pys (80)
> Patch Links:
> 
> https://github.com/coder-alpha/DesiTV.bundle/pull/3.patch
> https://github.com/coder-alpha/DesiTV.bundle/pull/3.diff
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.

hussamnasir committed:
Enable updater to pull automatically from my github fork.

hussamnasir committed:
Fixed missing Colors Channel

prasantsadangi commented:
Thnx...looks clean...!! 👍 👍

hussamnasir committed:
Update ServiceInfo.plist

hussamnasir committed:
Update README.md

Update credits and Known issues sections

hussamnasir committed:
Update desitashan.py

Had a double slash in the episode URL which plex and the site does not like. Just fixed that.

hussamnasir committed:
Update __init__.py

Enabled the old updater script again.

hussamnasir committed:
Revert "Update __init__.py"

This reverts commit cf559d47bc5708e4198ed4254edc5a63bf8ca3f9.

hussamnasir committed:
Enabled Desitvbox again becasue desitashan is having DB problems.
Enabled just the same sources that wored for desi-tashan.

hussamnasir committed:
Small fixes and re-enabled desitvbox for the same sources

hussamnasir committed:
missied commenting out desirulez since i introduced syntax errors.
